docs/sp-add-ins/add-in-permissions-in-sharepoint.md

```diff
@@ -295,7 +295,7 @@ SharePoint Add-ins that are installed to SharePoint are granted permissions when
  
 When you are developing an add-in or troubleshooting an add-in, there may be occasions when you want to change, or regrant, the permissions of an add-in that has already been installed. You can do so with these steps:
 
-1. Go to `http://<SharePointWebSite>/_layouts/15/AppInv.aspx`, where  _<SharePointWebSite>_ is the URL of the website where the add-in is installed. Be careful not to add any query parameters on the URL. The form you need only appears on this page if the URL is exactly as shown.
+1. Go to `http://<SharePointWebSite>_layouts/15/AppInv.aspx`, where  _\<SharePointWebSite\>_ is the URL of the website where the add-in is installed. Be careful not to add any query parameters on the URL. The form you need only appears on this page if the URL is exactly as shown.

```

Comment:
Incorrect URL, the forward slack is correct.

You can escape the `<SharePointWebSite>` with backslashes, or replace the <> wth {}